app/components/embed/media_tag_component.rb

```diff
@@ -87,7 +87,27 @@ def enabled_streaming_sources
     def transcript
       return unless render_captions?
 
-      tag.track(src: file.vtt.file_url, kind: 'captions', srclang: 'en', label: 'English')
+      # A video clip may have multiple caption files in different languages.
+      # We want to enable the user to select from any of these options.
+      # We also want the different language options to be listed alphabetically.
+      safe_join(
+        file.vtt.sort_by { |cfile| caption_language(cfile.language) }.map do |caption_file|
```

Comment:
How about extracting this first part to a method: `sorted_text_tracks`?

Reply:
Sounds good